db-engine/src/main/java/com/splicemachine/db/impl/sql/conn/GenericLanguageConnectionContext.java

```diff
@@ -3855,6 +3856,11 @@ private String formatLogStmt(String statement) {
             if (maxStatementLogLen >= 0 && maxStatementLogLen < statement.length()) {
                 subStatement = statement.substring(0, maxStatementLogLen) + " ... ";
             }
+            subStatement = StringUtils.maskMessage(subStatement, StringUtils.maskPasswordPattern,StringUtils.maskString);
```

Comment:
I'm a little bit worried about the cost of this function, since it's parsing the full statement, specially with very long statements. Can we run some benchmarks to make sure this is not affecting performance for OLTP workloads?

Reply:
The performance concern will be covered Jira DB-8828